### PR TITLE
Update doc for changes in wrappers

### PR DESCRIPTION
**Pull request by clostao**
### **User description**
Changes in #161 were not reflected in the doc. This PR updates it

___

### **PR Type**
Documentation

___

### **Description**
- Updated the `README.md` files for both `auto-dag-data` and `auto-drive` packages to reflect recent changes in function names and usage.
- Introduced new examples demonstrating the use of `MemoryBlockstore` and updated function names for creating IPLD DAGs.
- Added detailed sections on handling observables, uploading files from different interfaces, and downloading files using the `auto-drive` API.
- Improved documentation to enhance usability and understanding of the new features and changes.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Update function names and examples in README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/auto-dag-data/README.md

<li>Updated function names for creating IPLD DAGs.<br> <li> Added examples for new function usage.<br> <li> Introduced <code>MemoryBlockstore</code> in examples.<br>

</details>

  </td>
  <td><a href="https://github.com/autonomys/auto-sdk/pull/167/files#diff-ad6bcff069f61e89c8a449a5910a4d2e213d8ecc7b0a25ebe666847e2ec853e4">+26/-16</a>&nbsp; </td>

</tr>

<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Revise examples and add new sections in README</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

packages/auto-drive/README.md

<li>Revised examples for file upload and download.<br> <li> Added sections for handling observables and custom interfaces.<br> <li> Updated API initialization examples.<br>

</details>

  </td>
  <td><a href="https://github.com/autonomys/auto-sdk/pull/167/files#diff-aa6ce24aa92e54fce14c6035c8946b4c6bccee2f80288bd38feda74e6a55594c">+171/-20</a></td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Review by marc-aurele-besner — DISMISSED** [review]

**Review by marc-aurele-besner — APPROVED** [review]

File: packages/auto-dag-data/README.md [hunk 1]
```diff
@@ -36,33 +36,40 @@ yarn add @autonomys/auto-dag-data
 
 ### Creating an IPLD DAG from a File
 
-To create an IPLD DAG from a file, you can use the `createFileIPLDDag` function:
+To create an IPLD DAG from a file, you can use the `processFileToIPLDFormat` function:
 
 ```typescript
-import { createFileIPLDDag } from '@autonomys/auto-dag-data'
+import { processFileToIPLDFormat } from '@autonomys/auto-dag-data'
+import { MemoryBlockstore } from 'blockstore-core/memory'
```
Comment (marc-aurele-besner) [inline packages/auto-dag-data/README.md:43]: if we use blockstore-core inside this package we should expose all functions needed to not need extra dependencies to use this package

**Comment by marc-aurele-besner** [comment]
What is the blocker to merge this PR?

File: packages/auto-dag-data/README.md (hunk 1 above)
Comment (clostao) [reply packages/auto-dag-data/README.md:43]: It's just a devDependency so I'd prefer not to add it to `dependencies`

**Comment by clostao** [comment]
> What is the blocker to merge this PR?

No one now

**Comment by clostao** [comment]
The skip on workflow led to not be mergeable I tried to close & reopen but didn't work either. I'd need another reviewal

**Review by marc-aurele-besner — APPROVED** [review]

**Pull request merged**